Gds/src/fprime_gds/common/pipeline/encoding.py

```diff
@@ -50,15 +50,19 @@ def setup_coders(self, dictionaries, distributor, sender):
 
         :param dictionaries: a dictionaries handling object holding dictionaries
         :param distributor: distributor of data to register to
+        :param sender: TODO
```

Comment:
`sender` is used to send the bytes created by the command encoder

Reply:
I added this to the doc comment